routes.yaml

```diff
@@ -2,8 +2,7 @@ app_name: officer-filing-api
 group: api    
 weight: 900
 routes:
-  1: ^/officer-filing
+  1: ^/officer-filing/healthcheck
```

Comment:
Not quite clear on why this change is required?

Reply:
Tightening up access. The only thing on the /officer-filing path is the healthcheck now.